src/Microsoft.ComponentDetection.Detectors/maven/MavenParsingUtilities.cs

```diff
@@ -1,27 +1,29 @@
 ﻿using System;
+using System.Runtime.CompilerServices;
 using Microsoft.ComponentDetection.Contracts;
+using Microsoft.ComponentDetection.Contracts.BcdeModels;
 using Microsoft.ComponentDetection.Contracts.TypedComponent;
 
 namespace Microsoft.ComponentDetection.Detectors.Maven
 {
     public static class MavenParsingUtilities
     {
-        public static (DetectedComponent Component, bool? IsDevelopmentDependency) GenerateDetectedComponentFromMavenString(string key)
+        public static (DetectedComponent Component, bool? IsDevelopmentDependency, DependencyScope? dependencyScope) GenerateDetectedComponentAndIsDeveDependencyAndDependencyScope(string key)
```

Comment:
nit: I would probably name this something like `GenerateDetectedComponentAndMetadtaFromMavenString`, but either way `AndIsDeveDependency` has an extra e

Reply:
Also, you're using different casing patterns for the return values here

Reply:
renamed to `GenerateDetectedComponentAndMetadtaFromMavenString`.